### DISCUSSION TITLE
npm run test建立websocket失败，尝试更换版本也无效

### DISCUSSION THREAD

**Discussion by Lee0528Hub** [Tsuk1ko/CQ-picfinder-robot#350] [问题求助] [answered]
v1.0.0-rc2，v1.0.0-rc3，v1.0.0-beta7都试过了，没有修改config文件除QQ外的配置。
提前谢谢大佬们了。
npm run test提示
![02](https://user-images.githubusercontent.com/110772808/184507018-648f6800-8d36-4838-88ff-7d7b7523af58.png)
config.yml
![01](https://user-images.githubusercontent.com/110772808/184506993-28e46d29-7d56-492b-85a9-e2751698aaba.png)
config.jsonc
![03](https://user-images.githubusercontent.com/110772808/184507013-7d6d1999-f32e-4618-8430-a0428ef45d74.png)

**Comment by Tsuk1ko** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/24877906/184507092-686eee5a-4b0f-4e81-a19b-ca37bc0608c3.png)

**Comment by Lee0528Hub** [comment]
已解决，小白尝试配置机器人，实在不好意思耽误大佬晚上时间了。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Tsuk1ko/cq-picsearcher-bot"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月14日(星期天) 凌晨2:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Tsuk1ko/cq-picsearcher-bot] npm run test建立websocket失败，尝试更换版本也无效 (Discussion #350)

**Reply by xzhxzhxzh123** [reply]
请问楼主是把两个端口号改成一样的了吗？具体改成多少了？

**Reply by Miuzarte** [reply]
> 请问楼主是把两个端口号改成一样的了吗？具体改成多少了？

多少都行 不被占用就好，随便按五个数字上去

**Reply by Lee0528Hub** [reply]
> 请问楼主是把两个端口号改成一样的了吗？具体改成多少了？

不被占用的端口就可以，两个文件中的端口号要一致

**Reply by Tsuk1ko** [reply]
也不是随便五个数字，最大 `65535`（

直接把 go-cqhttp 中的 ws 端口抄到项目配置中即可
